src/app/vars/easing.css

```diff
@@ -0,0 +1,3 @@
+:root {
+  --easing-standard: cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
This is material design standard easing btw https://material.io/design/motion/speed.html#easing - I quite like their docs on motion